Source/Scene/ShadowMap.js

```diff
@@ -939,7 +893,11 @@ define([
             if (enterFreezeFrame) {
                 // Recreate debug camera when entering freeze frame mode
                 shadowMap._debugCameraFrustum = shadowMap._debugCameraFrustum && shadowMap._debugCameraFrustum.destroy();
-                shadowMap._debugCameraFrustum = createDebugFrustum(shadowMap._sceneCamera, Color.CYAN);
+                shadowMap._debugCameraFrustum = new DebugCameraPrimitive({
+                    camera : shadowMap._sceneCamera,
+                    color : Color.CYAN,
+                    updateOnChange : false
```

Comment:
I would leave it in because anyone who wants to visualize a camera that's moving will need this. It just happens that right now our only cases are seeing the scene camera in freeze frame.